src/Aspire.Hosting.MySql/MySqlBuilderExtensions.cs

```diff
@@ -248,20 +248,17 @@ private static string WritePhpMyAdminConfiguration(IEnumerable<MySqlServerResour
         writer.WriteLine();
         foreach (var mySqlInstance in mySqlInstances)
         {
-            if (mySqlInstance.PrimaryEndpoint.IsAllocated)
```

Comment:
Can we make the same change elsewhere? Line 126 above. And in PostgreSQL and Redis?

Reply:
I'll keep Redis and the test for another PR.